graphium/config/_loader.py

```diff
@@ -366,6 +367,7 @@ def load_trainer(
             seed=config["constants"]["seed"],
             model_name=config["constants"]["name"],
             gradient_accumulation=config["trainer"]["trainer"].get("accumulate_grad_batches", None),
+            precision=config["trainer"]["trainer"]["precision"]
```

Comment:
To handle when precision is not given

```suggestion
            precision=config["trainer"]["trainer"].get("precision")
```